Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:latest
+
+WORKDIR /app
```

Comment:
Can you take a look at `lnd`'s main Dockerfile (in the root directory of the repo) and use that as a template please? That has a few things that are better suited for production use and also produce smaller images.